docs/blog/001-train-custom-dataset/README.md

```diff
@@ -0,0 +1,245 @@
+# How to Train Your Custom Dataset with Anomalib
+<div align="center">
+<img src="docs/blog/../../images/anomalib-wide-blue.png" width="600px">
+</div>
+
+## Introducing Anomalib
+In early 2022, a group of Intel AI Researchers released a ground-breaking Deep Learning Library named Anomalib. The library provides state-of-the art algorithms for Anomaly Detection on Image datasets. Anomalib serves to be a thorough end-to-end solution, offering many features to achieve the highest accuracy model while also providing inference deployment code with Intel’s OpenVino Toolkit.
```

Comment:
Maybe we should be a bit more humble here. Ground-breaking may be a bit of an overstatement.

Reply:
Addressed